assets/messages.yml

```diff
@@ -376,6 +421,31 @@ messages:
         *Thank you for your report!*
         We're actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
 
+        %quick_links%
+      fillname:
+        - Enter parent issue ID
+  duplicate-resolved:
+    - project:
+        - bds
+        - mc
+        - mcapi
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Duplicate (resolved)
+      shortcut: rdup
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *%s%*, so this ticket is being resolved and linked as a *duplicate*.
+
+        That ticket has already been resolved. Please take a look at it to learn more.
+
+        %search_for_dupes%
+
         %quick_links%
       fillname:
         - Enter parent issue ID
```

Comment:
Why do we need that message? We already have individual messages for fixed, tech support, wai and won't fix.

Other resolutions are:
* Incomplete, but we don't use Incomplete tickets as a parent (or otherwise reopen it)
* Cannot Reproduce, but that either means it's fixed, or it should be reopened
* AR - might make sense to add a message for that saying something like "... but the parent is currently missing vital information. Please do take a look and see if you can help out, in order to get this bug fixed as soon as possible"
* Done - is not used anymore afaik.

Reply:
In theory we are missing one for "Invalid", though this message might not be that useful for this either?

Reply:
Well, we have one for tech support. Account issues obv. don't get linked, and a duplicate feature request doesn't happen often, and tbh I don't bother searching for a parent in this case and just resolve as invalid.